src/main/steps/birth-father/fatherSequence.test.ts

```diff
@@ -0,0 +1,110 @@
+import { YesNoUnsure, YesOrNo } from '../../app/case/definition';
```

Comment:
file could be renamed to birthFatherSequence just to inline with other sequence file names